core/src/main/java/org/apache/accumulo/core/conf/AccumuloConfiguration.java

```diff
@@ -209,13 +209,19 @@ static public long getMemoryInBytes(String str) {
         case 'B':
           multiplier = 0;
           break;
+        case '%':
```

Comment:
I would rather not create another `PropertyType`. If the defaults are fixed memory, users probably won't switch to percentage.  If you are worried about certain properties, I could add some text to the description of those properties that fixed memory amounts should be used.  Even if we limit percentage, users can still misconfigure things by setting fixed memory values that are too high or low for that property.